app/src/main/java/com/example/campusguide/directions/indoor/SelectAccessibilityOptionsDialogFragment.kt

```diff
@@ -0,0 +1,88 @@
+package com.example.campusguide.directions.indoor
+
+import android.app.AlertDialog
+import android.app.Dialog
+import android.graphics.Color
+import android.graphics.drawable.ColorDrawable
+import android.os.Bundle
+import android.widget.CheckBox
+import androidx.fragment.app.DialogFragment
+import com.example.campusguide.Accessibility
+import com.example.campusguide.DirectionsActivity
+import com.example.campusguide.R
+
+/**
+ * Fragment that contains a dialog window prompting the user to
+ * select the accessibility methods they would like to use for
+ * indoor pathfinding.
+ */
+class SelectAccessibilityOptionsDialogFragment(
+    private val directionsActivity: DirectionsActivity
+) : DialogFragment() {
+
+    override fun onCreateDialog(savedInstanceState: Bundle?): Dialog {
+        val inflater = requireActivity().layoutInflater
+        val view = inflater.inflate(R.layout.accessibility_select_in_directions, null)
+
+        // set title of dialog box
+        val builder: AlertDialog.Builder = AlertDialog.Builder(directionsActivity)
+            .setTitle("Select one or more options")
+
+        builder.setView(view)
+        builder.create().window?.setBackgroundDrawable(ColorDrawable(Color.TRANSPARENT))
+
+        builder.setPositiveButton("Confirm") { _, _ ->
+            handleOKSelected()
+        }.setNegativeButton("Cancel", null)
```

Comment:
Use the string constants `CONFIRM_CHOICE` and `CANCEL_CHOICE` for the button setters.